src/tests/unit/tests/bug-filing/azure-devops/__snapshots__/azure-boards-bug-filing-service.test.tsx.snap

```diff
@@ -0,0 +1,30 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Need to rename folder from `\src\tests\unit\tests\bug-filing\azure-devops\` to `azure-boards`

Reply:
updated